platformio/cli.py

```
@@ -18,7 +18,7 @@
 import click
 
 
-class PlatformioCLI(click.MultiCommand):
+class PlatformioCLI(click.Group):
     leftover_args = []
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)